gslib/tests/test_signurl.py

```diff
@@ -109,6 +110,19 @@ def testSignUrlInvalidDurationWithUseServiceAccount(self):
     self.assertIn('CommandException: Max valid duration allowed is 12:00:00',
                   stderr)
 
+  @unittest.skipUnless(not HAVE_CRYPTO, 'signurl requires crypto to decode .p12 keys')
```

Comment:
Have you confirmed that the package is present in the test environment?

Reply:
My point was if our tests only ever run in an environment without crypto or with crypto, a lot of the behavior relating to .p12 files won't ever be verified as either the tests that require crypto will silently be skipped, or the tests that require the absence of crypto will be silently skipped. Have you confirmed that there are kokoro builds, or other parts of our testing infrastructure, that we run this on that contain crypto and others that don't?

Reply:
I see, I've raised a CL for adding cryptography into test environment!

Reply:
Approved!